.github/workflows/packaging.yml

```diff
@@ -0,0 +1,89 @@
+name: packaging
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  package:
+    # Skip pull request jobs when the source branch is in the same
+    # repository.
+    if: |
+      github.event_name == 'push' ||
+      github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - { os: 'debian', dist: 'stretch' }
+          - { os: 'debian', dist: 'buster' }
+          - { os: 'debian', dist: 'bullseye' }
+          - { os: 'el', dist: '7' }
+          - { os: 'el', dist: '8' }
+          - { os: 'fedora', dist: '30' }
+          - { os: 'fedora', dist: '31' }
+          - { os: 'fedora', dist: '32' }
+          - { os: 'fedora', dist: '33' }
+          - { os: 'fedora', dist: '34' }
+          - { os: 'fedora', dist: '35' }
+          - { os: 'fedora', dist: '36' }
+          - { os: 'ubuntu', dist: 'xenial' }
+          - { os: 'ubuntu', dist: 'bionic' }
+          - { os: 'ubuntu', dist: 'focal' }
+          - { os: 'ubuntu', dist: 'groovy' }
+          - { os: 'ubuntu', dist: 'hirsute' }
+          - { os: 'ubuntu', dist: 'impish' }
+          - { os: 'ubuntu', dist: 'jammy' }
+
+    env:
+      OS: ${{ matrix.platform.os }}
+      DIST: ${{ matrix.platform.dist }}
+
+    steps:
+      - name: Clone the module
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Clone the packpack tool
+        uses: actions/checkout@v2
+        with:
+          repository: packpack/packpack
+          path: packpack
+
+      - name: Fetch tags
+        # Found that Github checkout Actions pulls all the tags, but
+        # right it deannotates the testing tag, check:
+        #   https://github.com/actions/checkout/issues/290
+        # But we use 'git describe ..' calls w/o '--tags' flag and it
+        # prevents us from getting the needed tag for packages version
+        # setup. To avoid of it, let's fetch it manually, to be sure
+        # that all tags will exist always.
+        run: git fetch --tags -f
+
+      - name: Create packages
+        run: ./packpack/packpack
+
+      - name: Deploy packages
+        # We need this step to run only on push with tag.
+        if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
+        env:
+          RWS_URL_PART: https://rws.tarantool.org/tarantool-modules
+          RWS_AUTH: ${{ secrets.RWS_AUTH }}
+          PRODUCT_NAME: tarantool-http
+        run: |
+          CURL_CMD="curl -LfsS \
+            -X PUT ${RWS_URL_PART}/${OS}/${DIST} \
+            -u ${RWS_AUTH} \
+            -F product=${PRODUCT_NAME}"
+
+          for f in $(ls -I '*build*' -I '*.changes' ./build); do
```

Comment:
Ignore `*build*`, but include `./build`. Hm. Can you clarify this logic, please? Will not we ignore `./build`? Is it for `./build/build.log`? If so, maybe it is better to do `-I '*.log'`?

Reply:
I see lots of stuff named using *build*. Probably, it's a good way to use `ls` in this case.
```bash
build
├── build.log
├── prebuild.sh
├── tarantool-http-1.2.0.10-1.el7.src.rpm
├── tarantool-http-1.2.0.10-1.el7.x86_64.rpm
├── tarantool-http-dbgsym_1.2.0.10-1_amd64.ddeb
├── tarantool-http-debuginfo-1.2.0.10-1.el7.x86_64.rpm
├── tarantool-http_1.2.0.10-1.debian.tar.xz
├── tarantool-http_1.2.0.10-1.dsc
├── tarantool-http_1.2.0.10-1_amd64.build
├── tarantool-http_1.2.0.10-1_amd64.buildinfo
├── tarantool-http_1.2.0.10-1_amd64.changes
├── tarantool-http_1.2.0.10-1_amd64.deb
└── tarantool-http_1.2.0.10.orig.tar.xz -> tarantool-http-1.2.0.10.tar.xz
```
If I didn't get you, please, explain again.

Reply:
`-I build.log -I prebuild.sh -I '*.build' -I '*.buildinfo'` would be self-descriptive IMHO. `*build*` looks cryptic.

Reply:
As we need only `deb` and `rpm` files, probably it's better to do like this:
```bash
for f in $(cd build; ls *{rpm,deb}); do
  CURL_CMD+=" -F $(basename ${f})=@./build/${f}"
done
```

Reply:
1. I would prefer `*.foo` over `*foo` -- less space for mistakes.
2. Will RWS accept `*.ddeb` (it is matched by the `*deb` glob)? See https://github.com/tarantool/rws/issues/13.
3. Shouldn't we store `*.dsc` files? RWS accepts them at least: https://github.com/tarantool/mkrepo/pull/61.
4. Here I don't know, just question: shouldn't we store debian source tarballs (`*.debian.tar.[gx]z`, `*.orig.tar.[gx]z`)?

Reply:
I’ll give you my 2 cents.
- we do not publish `*.ddeb` packages and I see no reason to start doing that. 
- we also publish `tar.xz`, `debian.tar.xz`, `orig.tar.xz` and `dsc` files. I suggest do it the same way we publish other packages, with the same set of files ([example](https://rws.tarantool.org/release/modules/ubuntu/pool/groovy/main/t/tarantool-queue))

Reply:
Okay, thanks! This solution seems to be complete and descriptive:
```bash
for f in $(cd build; ls *.(deb|rpm|dsc|tar.xz)); do
      CURL_CMD+=" -F $(basename ${f})=@./build/${f}"
done
```

Reply:
@Totktonada, what do you think about this solution? `shopt -s nullglob` allows to ignore not-matched patterns, and the cycle looks clear. The `woking-directory` is set in the job manifest.
```bash
shopt -s nullglob
for f in *.deb *.rpm *.dsc *.tar.xz; do
    CURL_CMD+=" -F $(basename ${f})=@${f}"
done
```

Reply:
I'm ok. The only comment: it is not a mistake that I said `.tar.[gx]z`, not just `.tar.xz`. I don't remember, when precisely it was changed, but I remember this PR: https://github.com/tarantool/mkrepo/pull/15.

Reply:
added *.tar.gz